#! /bin/sh /usr/share/dpatch/dpatch-run

## 01_Makefile-CFGDIR.dpatch by Thomas Schmidt <tschmidt@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Allow default config dir to be set as define.

@DPATCH@
diff -urNad vdr-1.3.23/Makefile /tmp/dpep.VCRgia/vdr-1.3.23/Makefile
--- vdr-1.3.23/Makefile	2005-02-13 11:13:45.000000000 +0100
+++ /tmp/dpep.VCRgia/vdr-1.3.23/Makefile	2005-04-09 21:28:19.430297680 +0200
@@ -23,6 +23,7 @@
 PLUGINLIBDIR= $(PLUGINDIR)/lib
 
 VIDEODIR = /video
+CFGDIR ?= $(VIDEODIR)
 
 DOXYGEN  = /usr/bin/doxygen
 DOXYFILE = Doxyfile
@@ -69,6 +70,7 @@
 DEFINES += -D_GNU_SOURCE
 
 DEFINES += -DVIDEODIR=\"$(VIDEODIR)\"
+DEFINES += -DCFGDIR=\"$(CFGDIR)\"
 DEFINES += -DPLUGINDIR=\"$(PLUGINLIBDIR)\"
 
 # The version number of VDR (taken from VDR's "config.h"):
diff -urNad vdr-1.3.23/vdr.c /tmp/dpep.VCRgia/vdr-1.3.23/vdr.c
--- vdr-1.3.23/vdr.c	2005-03-20 11:58:59.000000000 +0100
+++ /tmp/dpep.VCRgia/vdr-1.3.23/vdr.c	2005-04-09 21:28:19.431297528 +0200
@@ -381,7 +381,7 @@
   // Configuration data:
 
   if (!ConfigDirectory)
-     ConfigDirectory = VideoDirectory;
+     ConfigDirectory = CFGDIR;
 
   cPlugin::SetConfigDirectory(ConfigDirectory);
   cThemes::SetThemesDirectory(AddDirectory(ConfigDirectory, "themes"));
